python/tank/authentication/resources/login_dialog.ui

```diff
@@ -45,13 +48,13 @@ QLineEdit, QComboBox
     border: 1px solid rgb(54, 60, 66);
     border-radius: 2px;
     padding: 5px;
-     font-size: 12px;
+    font-size: 12px;
 }
 
 QComboBox
 {
-    margin-left: 3;
-    margin-right: 3
+    margin-left: 3px;
```

Comment:
Moving to px is usually not great for high dpi screens on Windows and on Linux. It's better to use points.

Reply:
Good catch!

However, this is out of context here as I am only cleaning up the code. No unit in QR4-5 means pixel -> https://doc.qt.io/qt-5/stylesheet-reference.html#length

Would you mind creating a new Jira for this, please? All this CSS needs to be reviewed anyway and make sure we use the same one between all widgets (tk-core, tk-desktop, framework-qtwidget, ...)